### PR TITLE
add some UTC clarification for timestamps

### PR DESCRIPTION
**Pull request by gregors**
Add some documentation that clarifies UTC requirements pertaining to issue https://github.com/elixir-ecto/myxql/issues/188

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +149,20 @@ shouldn't be used.
...
+UTC mode will need to be manually enabled.
+
+```elixir
+MyXQL.start_link(after_connect: &MyXQL.query!(&1, "SET time_zone = '+00:00'"))
+```
+or when configuring at the Ecto.Repo level
+
+```elixir
+after_connect: {MyXQL, :query!, ["SET time_zone = '+00:00'", []]}
+```
```
Comment (wojtekmach) [inline README.md:164]:
````suggestion
When using `DateTime` type, MyXQL assumes the server is configured with the UTC time
zone. If that is not the case, set it manually with `MyXQL.start_link/1` and
`:after_connect` option:

```elixir
MyXQL.start_link(after_connect: &MyXQL.query!(&1, "SET time_zone = '+00:00'"))
```

or when configuring `Ecto.Repo`:

```elixir
config :myapp, MyApp.Repo,
  after_connect: {MyXQL, :query!, ["SET time_zone = '+00:00'", []]}
```
````

**Review by wojtekmach — APPROVED** [review]
Thanks for looking into it!

**Comment by gregors** [comment]
@wojtekmach - I committed your suggestions. Just making sure you don't forget about this small issue

**Comment by wojtekmach** [comment]
Thanks for the PR and reminder!

**Pull request merged**